.github/workflows/jdl.yml

```diff
@@ -98,7 +98,7 @@ jobs:
       - name: 'SETUP: setup graalvm'
         uses: graalvm/setup-graalvm@v1
         with:
-          version: '22.0.0.2'
+          version: '22.1.0'
```

Comment:
22.2.0 is out now. https://github.com/graalvm/graalvm-ce-builds/releases/tag/vm-22.2.0